pandapower/pd2ppc_zero.py

```diff
@@ -421,11 +422,11 @@ def _add_ext_grid_sc_impedance_zero(net, ppc):
 
     # ext_grid zero sequence impedance
     if case == "max":
-        x0_grid = net.ext_grid["x0x_%s" % case].values * x_grid
-        r0_grid = net.ext_grid["r0x0_%s" % case].values * x0_grid
+        x0_grid = net.ext_grid[is_egs]["x0x_%s" % case].values * x_grid
```

Comment:
To me ll. 425/426 and 428/429 look exactly the same. Could we just delete one part of both?

Reply:
I also see it this way